wcfsetup/install/files/lib/system/style/StyleCompiler.class.php

```diff
@@ -51,7 +51,44 @@ class StyleCompiler extends SingletonFactory {
 	 */
 	protected function init() {
 		require_once(WCF_DIR.'lib/system/style/scssphp/scss.inc.php');
-		$this->compiler = new Compiler();
+		$this->compiler = new class extends Compiler {
```

Comment:
XXX This violates every formatting rule we have. Clean up and move into separate class files.